docker-compose.yml

```diff
@@ -2,23 +2,25 @@ version: '3.3'
 services:
   ldes-server:
     env_file:
-      - ./config.local.env
-    container_name: ldes-server
+      - ./docker-compose/config.local.env
+    container_name: ldes.server
```

Comment:
out of curiosity, what's the reasoning behind the dot?

Reply:
The dot is irrelevant, but the containers should have names that distinguishes them from containers that use the same image in other projects (outside of vsds). Do you prefer an underscore ?

Reply:
no, it's okay :-)